testbench/common.py

```diff
@@ -338,6 +341,30 @@ def enforce_patch_override(request):
         testbench.error.notallowed(context=None)
 
 
+def __get_default_response_fn(data):
+    return data
+
+
+def handle_retry_test_instruction(database, request, method):
```

Comment:
Are we splitting this function and checking in the `error_code_matches` part first?

Reply:
I may be misinterpreting your question.  The function looks a bit awkward right now because it just handles non-streaming errors.  It will get bigger eventually, but I am being lazy and not adding this stuff until I strictly need it:

https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/storage/emulator/utils/common.py#L380-L424

Reply:
Yep that was my question; I was just following up on handling retry instructions for connection errors and streaming errors. Your plan SGTM 👍